src/main/java/frc/robot/subsystems/elevator/ElevatorIONeo.java

```diff
@@ -30,7 +30,7 @@ public class ElevatorIONeo implements ElevatorIO {
   private final DigitalInput topLimitSwitch;
 
   private final double encoderLowerLimit = 0.0;
-  private final double encoderUpperLimit = 280.0;
+  private final double encoderUpperLimit = 280.0 / 3;
```

Comment:
We desperately need human-understandable constants. This should be inches, not some arbitrary value.